File: extract_logo.py
import re

file_path = "C:/Users/saro_/Desktop/tarjeta_sagitario_codigo_completo.html"

with open(file_path, "r", encoding="utf-8") as f:
    content = f.read()

# Find the base64 image source
match = re.search(r'src="(data:image/jpeg;base64,[^"]+)"', content)
if match:
    base64_data = match.group(1)
    print("Successfully found base64 logo data!")
    print(f"Length: {len(base64_data)} characters")
    
    # Save base64 to a temporary text file so we can read it easily
    with open("logo_base64.txt", "w", encoding="utf-8") as out:
        out.write(base64_data)
    print("Saved logo to logo_base64.txt")
else:
    print("Could not find base64 logo data in the file!")

File: extract_logo.js
const fs = require('fs');

const filePath = 'C:/Users/saro_/Desktop/tarjeta_sagitario_codigo_completo.html';

try {
  const content = fs.readFileSync(filePath, 'utf8');
  const match = content.match(/src="(data:image\/jpeg;base64,[^"]+)"/);
  
  if (match) {
    const base64Data = match[1];
    console.log("Successfully found base64 logo data!");
    console.log(`Length: ${base64Data.length} characters`);
    
    fs.writeFileSync('logo_base64.txt', base64Data, 'utf8');
    console.log("Saved logo to logo_base64.txt");
  } else {
    console.log("Could not find base64 logo data in the file!");
  }
} catch (err) {
  console.error("Error reading file:", err.message);
}
